src/web/app/pages-help/instructor-help-page/instructor-help-page.component.html

```diff
@@ -1,49 +1,74 @@
+<div id="body" style="width:1100px;">
+  <div tmScrollSpy>
+    <h1 class="color-orange" id="body"> Help for Instructors</h1>
+    <div>
       <p>
-        Browse questions by topic:
+        Have questions about how to use TEAMMATES? This page answers some frequently asked questions.
+        <br> If you are new to TEAMMATES, our <a routerLink="../getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
+        <br> If you have any remaining questions, don't hesitate to <a href="mailto:{{ supportEmail }}">email us</a>. We respond within 24 hours.
       </p>
-      <ul>
-        <li>
-          <a href="#students">Students</a>
+      <div class="row">
+        <div class="offset-md-2 col-md-8">
+          <div class="input-group input-group-lg">
+            <input type="text" [(ngModel)]="searchTerm" class="form-control input-lg" placeholder="How can we help?" (keyup.enter)="search()">
+            <button class="btn btn-primary btn-lg" (click)="search()"><i class="fas fa-search"></i></button>
+            <button class="btn btn-danger btn-lg" (click)="clear()"><i class="fas fa-times"></i></button>
+          </div>
+        </div>
+      </div>
+      <hr/>
+      <div id="students">
+        <tm-instructor-help-students-section [key]="key"> 1
+        </tm-instructor-help-students-section>
+      </div>
+      <div id="courses">
+        <tm-instructor-help-courses-section [key]="key"> 2
+        </tm-instructor-help-courses-section>
+      </div>
+      <div id="sessions">
+        <tm-instructor-help-sessions-section [key]="key"> 3
+        </tm-instructor-help-sessions-section>
+      </div>
+      <div id="questions">
+        <tm-instructor-help-questions-section [key]="key"> 4
+        </tm-instructor-help-questions-section>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="container-fluid" style="position: fixed; left: 1170px; top: 100px;">
+  <div class="section scrollspy">
+    <div id="topics" class="col-sm-3 col-4">
+      <p>
+        Browse by topic:
+        <br>
+      </p>
+      <ul class="nav nav-pills flex-column">
+        <li class="nav-item">
+          <a (click)="scrollTo('students')" [routerLink]="">Students</a>
+          <ng-container *ngIf="currentSection==='students'"></ng-container>
         </li>
-        <li>
-          <a href="#courses">Courses</a>
+        <li class="nav-item">
+          <a (click)="scrollTo('courses')" [routerLink]="">Courses</a>
+          <ng-container *ngIf="currentSection==='courses'"></ng-container>
         </li>
-        <li>
-          <a href="#sessions">Sessions</a>
+        <li class="nav-item">
+          <a (click)="scrollTo('sessions')" [routerLink]="">Sessions</a>
+          <ng-container *ngIf="currentSection==='sessions'"></ng-container>
         </li>
-        <li>
-          <a href="#questions">Questions</a>
+        <li class="nav-item">
+          <a (click)="scrollTo('questions')" [routerLink]="">Questions</a>
```

Comment:
Are all these `[routerLink]=""` necessary?

Reply:
Not really 'necessary', but they make the side nav links, well, links. Otherwise they would be plain text and the cursor would not change into the hand with the index finger out. I chose to make them links as a stylistic choice, but they would be functionally the same without the routerlinks.

Reply:
Try something like this instead:
`<a tabindex="0" role="button" (click)="scrollTo('questions')>Questions</a>`

Reply:
This doesn't change the cursor shape, and box appears around the last clicked link, but is otherwise passable. I can push the alternate versions if you recommend it, but links on similar sidenavs (like the one in this link https://angular.io/api/core/ViewChild) seem to match the existing version more.

Reply:
Let's go with your original one first then :)

Reply:
The issue with `[routerLink]=""` is that all your original query parameters would be lost after execution. Given that the help page is mostly static, it might not affect things much at this point. 

But let's keep this in mind in the event query parameters or dynamic links are related.

Reply:
Gotcha, thanks for taking the time out to explain :)